README.md

```
@@ -24,164 +24,94 @@ Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 
-
-## Install with [docker compose](https://github.com/docker/compose)
-
-1) At first, use composer to install this project.
-
-   ```
-   composer create-project -s dev contributte/apitte-skeleton
+   ```bash
+docker run -p 5432:5432 -e POSTGRES_PASSWORD=contributte -e POSTGRES_USER=contributte dockette/postgres:15
+# or
+docker run -p 3306:3306 -e MARIADB_ROOT_PASSWORD=contributte -e MARIADB_USER=contributte -e MARIADB_PASSWORD=contributte -e MARIADB_DATABASE=contributte mariadb:10.4
    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)